File: Remove Linked List Elements/RemoveLinkedListElements.java
/**
 * Definition for singly-linked list.
 * public class ListNode {
 *     int val;
 *     ListNode next;
 *     ListNode(int x) { val = x; }
 * }
 */
public class Solution {
    public ListNode removeElements(ListNode head, int val) {
        
        ListNode prev=head;
        
        if(prev==null){
            return null;
        }
        
        while(prev.next!=null){
            
            if(prev.next.val==val){
                prev.next=prev.next.next;
            }else{
                prev=prev.next;
            }
        }
        
        return head.val==val?head.next:head;
        
    }
}

File: Remove Linked List Elements/remove_linked_list_elements.py
# Definition for singly-linked list.
# class ListNode(object):
#     def __init__(self, x):
#         self.val = x
#         self.next = None

class Solution(object):
    def removeElements(self, head, val):
        """
        :type head: ListNode
        :type val: int
        :rtype: ListNode
        """
        if head==None:
            return None
        
        prev=head
        
        while prev.next!=None:
            if prev.next.val==val:
                prev.next=prev.next.next
            else:
                prev=prev.next
        return head if head.val!=val else head.next